### PR TITLE
Reorder accounting buttons

### PR DESCRIPTION
**Pull request by julianseeger**
resolves https://github.com/Qabel/qabel-desktop/issues/215
depends on / includes https://github.com/Qabel/qabel-desktop/pull/462

it's more an extract and move functionality from AccountingController to the IdentityContextMenuController instead of a reorder.

### CODE REVIEW

**Comment by thechauffeur** [comment]
This should not be merged into the current master, not before release.

**Comment by thechauffeur** [comment]
I mean even if working properly.

**Comment by julianseeger** [comment]
image

File: src/main/resources/ui.properties [hunk 1]
```diff
@@ -248,5 +249,6 @@ phoneInstructions=Please enter a valid phone numer, if you want to be found by o
   "Next". You can still add your phone number to your identity later. We will send you a \
   text message to confirm phone number.
 searchIndex=Search for contacts
+confirmIdentitiyDeletion=Do you really want to delete the identity {0}?
```
Comment (JoGir) [inline src/main/resources/ui.properties]: typo

File: src/main/resources/ui_de_DE.properties [hunk 2]
```diff
@@ -199,7 +199,7 @@ emailInstructions=Bitte geben Sie hier eine gültige E-Mail-Adresse ein, wenn Si
 finishWizard=Bitte beachten Sie, dass Ihre Identität aktuell nur auf diesem Gerät zur Verfügung steht. \
   Qabel oder Dritte haben keine Kopien auf Ihren Servern. Exportieren Sie zur Sicherheit \
   als Backup Ihre Identität als .qid und bewahren Sie diese sicher auf. \
-  Die Identität mit der Endung .qid enthält den privaten Schlüssel Ihrer Identität, \
+  Die Identität mit der ndung .qid enthält den privaten Schlüssel Ihrer Identität, \
```
Comment (JoGir) [inline src/main/resources/ui_de_DE.properties]: typo

File: src/main/resources/ui.properties (hunk 1 above)
Comment (julianseeger) [inline src/main/resources/ui.properties]: sry I don't see it o.O

File: src/main/resources/ui_de_DE.properties (hunk 2 above)
Comment (julianseeger) [inline src/main/resources/ui_de_DE.properties]: fixed in https://github.com/Qabel/qabel-desktop/pull/479/commits/53e7708427fe7cee67d56d8569663d81814012e0

File: src/main/resources/ui.properties (hunk 1 above)
Comment (julianseeger) [inline src/main/resources/ui.properties]: oh the typo is in the key, not in the value ... took me 12h o.O

**Comment by julianseeger** [comment]
@JoGir please review / merge

**Pull request merged**